pages/docs/disclosure/tabs.mdx

```diff
@@ -399,24 +399,23 @@ to pass state internally. Your custom `Tab` component must use
 
 ```jsx
 function CustomTabs() {
-  // 1. Reuse the styles for the Tab
-  const StyledTab = chakra('button', { themeKey: 'Tabs.Tab' })
 
   const CustomTab = React.forwardRef((props, ref) => {
-    // 2. Reuse the `useTab` hook
+    // 1. Reuse the `useTab` hook
     const tabProps = useTab({ ...props, ref })
     const isSelected = !!tabProps['aria-selected']
 
-    // 3. Hook into the Tabs `size`, `variant`, props
-    const styles = useStyles()
+    // 2. Hook into the Tabs `size`, `variant`, props
+    const styles = useMultiStyleConfig("Tabs", tabProps);
+
 
     return (
-      <StyledTab __css={styles.tab} {...tabProps}>
+      <Box __css={styles.tab} {...tabProps}>
```

Comment:
This would render `div` tags instead of `button` tags. Here's the old way:
<img width="1069" alt="CleanShot 2022-01-18 at 09 21 57@2x" src="https://user-images.githubusercontent.com/5396211/149898429-160526ed-506a-4602-bbe6-ff319b65d024.png">

And here's this implementation:
<img width="1063" alt="CleanShot 2022-01-18 at 09 22 52@2x" src="https://user-images.githubusercontent.com/5396211/149898482-9d043827-1a87-4941-a040-5ee813cd29a7.png">

Also there's no Pointer cursor when hovering the tabs. That could be confusing.

Reply:
Indeed! Fixing this asap :)